mockup/patterns/resourceregistry/js/builder.js

```diff
@@ -150,6 +154,16 @@ define([
               self.finished(true);
             }
           });
+        }else if($styles.length === config.less.length){
+          $styles.each(function(){$(this).remove();});
+
+          script = document.createElement('script');
+          script.setAttribute('type', 'text/javascript');
+          script.setAttribute('src', self.rview.options.data.lessModifyUrl);
```

Comment:
I looked at this code and it just seemed funny. At this point, wouldn't self.rview.options.data.lessModifyUrl be null/undefined? So why are we loading it this way?